snap/info.go

```diff
@@ -553,6 +553,26 @@ type TimerInfo struct {
 	Timer string
 }
 
+// RefreshModeType is the type for the "refresh-mode:" of a snap app
+type RefreshModeType string
+
+func (rm RefreshModeType) KillMode() string {
+	switch rm {
+	case "sigterm", "sighup", "sigusr1", "sigusr2":
+		return "process"
```

Comment:
I find it a bit strange to encode this here,  it's far way from the other places in wrappers that deal with the value of RefreshMode

Reply:
Yes!